### PR TITLE
DOC-638 Update rpk with 24.2.6. Minor text adjustment to rpk-redpanda-start

### PR DESCRIPTION
**Pull request by Deflaimun**
## Description

Update rpk with 24.2.6.

Adds some improvements to `rpk redpanda start`

## Page previews

[rpk redpanda start](https://deploy-preview-803--redpanda-docs-preview.netlify.app/current/reference/rpk/rpk-redpanda/rpk-redpanda-start/)

## Checks

- [ ] New feature
- [ ] Content gap
- [ ] Support Follow-up
- [x] Small fix (typos, links, copyedits, etc)

### CODE REVIEW

File: modules/reference/pages/rpk/rpk-redpanda/rpk-redpanda-start.adoc [hunk 1]
```diff
@@ -95,7 +95,7 @@ numbers, each with optional fraction and a unit suffix, such as
 
 |--tune |- |When present will enable tuning before starting Redpanda.
 
-|--well-known-io |string |The cloud provider and VM type, in the format <provider>:<vm type>:<storage type>.
+|--well-known-io |string |The cloud vendor and VM type, in the format <vendor>:<vm type>:<storage type>.
```
Comment (kbatuigas) [inline modules/reference/pages/rpk/rpk-redpanda/rpk-redpanda-start.adoc]:
```suggestion
|--well-known-io |string |The cloud vendor and VM type, in the format `<vendor>:<vm type>:<storage type>`.
```
From a quick glance of our docs - we don't typically use a hyphen inside placeholders in the rpk reference... do we?

**Review by kbatuigas — APPROVED** [review]
No blockers, but also going to note that there might be some text on the rest of the page that could be formatted (code fence) or copy edited (e.g. instances of `redpanda`)

File: modules/reference/pages/rpk/rpk-redpanda/rpk-redpanda-start.adoc (hunk 1 above)
Comment (Deflaimun) [reply modules/reference/pages/rpk/rpk-redpanda/rpk-redpanda-start.adoc]: Yeah that page needs a bit more love. I'll add some styling to it to better match our docs, even though they don't exist in the automation.

File: modules/reference/pages/rpk/rpk-redpanda/rpk-redpanda-start.adoc [hunk 2]
```diff
@@ -56,46 +56,91 @@ rpk redpanda start [flags]
...
+
+|--advertise-rpc-addr |string |The advertised RPC address 
 
-|--advertise-rpc-addr |string |The advertised RPC address (host:port).
+[,bash]
+----
+<host>:<port>
+----
 
 |--check |- |When set to false will disable system checking before
```
Comment (kbatuigas) [inline modules/reference/pages/rpk/rpk-redpanda/rpk-redpanda-start.adoc]:
```suggestion
|--check |- |When set to `false`, will disable system checking before
```

File: modules/reference/pages/rpk/rpk-redpanda/rpk-redpanda-start.adoc [hunk 3]
```diff
@@ -56,46 +56,91 @@ rpk redpanda start [flags]
...
-starting redpanda (default true).
+starting `redpanda` (default true).
```
Comment (kbatuigas) [inline modules/reference/pages/rpk/rpk-redpanda/rpk-redpanda-start.adoc]:
```suggestion
starting `redpanda` (default `true`).
```
Not sure if redpanda should be in backticks, seems like we could just use "Redpanda"?

File: modules/reference/pages/rpk/rpk-redpanda/rpk-redpanda-start.adoc [hunk 4]
```diff
@@ -56,46 +56,91 @@ rpk redpanda start [flags]
...
 
 |-h, --help |- |Help for start.
 
-|--install-dir |string |Directory where redpanda has been installed.
+|--install-dir |string |Directory where `redpanda` has been installed.
```
Comment (kbatuigas) [inline modules/reference/pages/rpk/rpk-redpanda/rpk-redpanda-start.adoc:86]: Not sure about the backticks here either, are we possibly referring to the binary..?

File: modules/reference/pages/rpk/rpk-redpanda/rpk-redpanda-start.adoc [hunk 5]
```diff
@@ -56,46 +56,91 @@ rpk redpanda start [flags]
...
 |--kafka-addr |strings |A comma-separated list of Kafka listener
-addresses to bind to (scheme://host:port\|name).
+addresses to bind to.
+
+[,bash]
+----
+<scheme>:<host>:<port>\|<name>
+----
 
 |--mode |string |Mode sets well-known configuration properties for
```
Comment (kbatuigas) [inline modules/reference/pages/rpk/rpk-redpanda/rpk-redpanda-start.adoc]:
```suggestion
|--mode |string |Sets well-known configuration properties for
```

**Review by kbatuigas — APPROVED** [review]
No blockers, just minor comments

File: modules/reference/pages/rpk/rpk-redpanda/rpk-redpanda-start.adoc (hunk 3 above)
Comment (Deflaimun) [reply modules/reference/pages/rpk/rpk-redpanda/rpk-redpanda-start.adoc]: used backticks as it's the redpanda process

File: modules/reference/pages/rpk/rpk-redpanda/rpk-redpanda-start.adoc (hunk 4 above)
Comment (Deflaimun) [reply modules/reference/pages/rpk/rpk-redpanda/rpk-redpanda-start.adoc:86]: redpanda process

**Review by r-vasquez — APPROVED** [review]

**Pull request merged**